CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# CONTRIBUTING GUIDELINES
+
+NovelRT is a community-driven project. Contribution is always welcome, and much appreciated.
+It is also how it'll be developed.
```

Comment:
Unsure if this line is really neccessary, as the statement right before this kind of states this, I feel?